test/specs/helpers.core.tests.js

```diff
@@ -264,6 +264,13 @@ describe('Chart.helpers.core', function() {
 		});
 	});
 
+	describe('_elementsEqual', function() {
+		it('should return false if arrays are not the same', function() {
```

Comment:
Should also return true if they are equal. I think a test or two more should be added.

Reply:
done